lib/data/repository/tasks/tasks_repository.dart

```diff
@@ -138,4 +138,10 @@ class TasksRepository with UpdatesStream<dynamic> implements TasksDataSource {
 
   @override
   Future<void> deleteAllData() => _cacheDataSource.deleteAllData();
+
+
+  @override
+  Future<void> updateTaskGroup(TaskGroup key, List<String> orderedTaskList) {
```

Comment:
Change the method signature to match the method in TaskDataSource.
If you want you can have another convenience method in TasksRepository that will change the task order and call updateTaskGroup.